src/components/autonomous/WaypointNav.js

```diff
@@ -18,6 +20,23 @@ function WaypointNav() {
     dispatch(requestWaypointNav(formJson));
   };
 
+  function grabFromClipboard () {
+    navigator.clipboard.readText().then(text => {
+      // Matches coordinates in the form of (-)*(.*), (-)*(.*)
+      // where * are numbers and () are optional, e.g. -0.2, 0
+      if(text.match("-?\\d+\\.?\\d*, -?\\d+\\.?\\d*")) {
+        const [lat, lon] = text.split(", ", 2);
+        setLat(lat);
+        setLon(lon);
+      } else {
+        alert("Clipboard contents do not match regex!");
```

Comment:
Super minor, but the users probably won't know what a regex is, or why that's important. Just say something like "Data in clipboard does not match the expected format! Format is: '<latitude>, <longitude>'" or something.